ducktape/tests/test.py

```diff
@@ -133,6 +134,9 @@ def _escape_pathname(s):
     return re.sub("^\.|\.$", "", s)
 
 
+CLUSTER_SIZE_KEYWORD = "num_nodes"
```

Comment:
I was imagining the concept of expected `num_nodes` as being something defined within/alongside `TestContext`, and having `ClusterUseMetadata` provide a dumb container for key->val pairs.

More practically, it helps avoid circular import since ducktape.mark._mark imports `TestContext`